front/app/containers/IdeasFeedPage/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+import { Box, colors, useBreakpoint } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+
+import useProjectBySlug from 'api/projects/useProjectBySlug';
+
+import GoBackButton from 'components/UI/GoBackButton';
+
+import IdeasFeedPageMeta from './IdeasFeedPageMeta';
+import TopicsSidebar from './TopicsSidebar';
+
+const IdeasFeedPage = () => {
+  const { slug } = useParams() as { slug: string };
+  const { data: project } = useProjectBySlug(slug);
+  const [selectedTopicId, setSelectedTopicId] = useState<string | null>(null);
```

Comment:
Could we store this in the URL instead of state? Feels important to be able to link to externally.